src/app/dts/dts.component.spec.ts

```diff
@@ -52,10 +55,10 @@ describe('DtsComponent', () => {
   });
 
   it('should display a list of templates by document type', done  => {
-    //const localTemplateItems = templates.items.filter(template => template.docType === 'enrollment');
-    spyOn(dtsService, 'getTemplates').and.returnValue(of(templatesEnrollment.items));
+    // reset the spy and expected result
+    dtsService.getTemplates = jasmine.createSpy().and.returnValue(of(templatesEnrollment.items));
 
-    component.getTemplates();
+    fixture.detectChanges();
 
     dtsService.getTemplates().subscribe( result => {
       expect(result.length).toEqual(2);
```

Comment:
This code is testing your mock but not really checking the actual component rendering. As a matter of fact, no assertions are run against your component in these two tests. That's something that should be done here, using `fixture.debugElement` to validate the HTML rendering.

Reply:
Good catch.  Reworked test to validate the grid content.